collect_app/src/main/java/org/odk/collect/android/views/WidgetAnswerText.kt

```diff
@@ -166,7 +167,9 @@ class WidgetAnswerText(context: Context, attrs: AttributeSet?) : FrameLayout(con
     }
 
     fun setError(error: String?) {
-        binding.textInputLayout.error = error
+        binding.textInputLayout.post {
```

Comment:
Why the `post` here?

Reply:
This new scenario needs some delay to clear the error after recreating the view.